ci/Conformance.md

```diff
@@ -0,0 +1,67 @@
+# Conformance results
+
+## To check for the existing nodes
+```
+$ kubectl get nodes
+```
+
+## Recommended host machine configuration
+1. RAM   : 32 GB
+2. vCPUs : 8  
+
+## Heptio Sonobuoy test
+The standard tool for running the conformance tests, Sonobuoy, is already installed while deploying the kubernetes cluster using snaps-orchestration.
+
+### Install Sonobuoy
```

Comment:
Put the install latest release section first and list it is (Recommend).

For the install latest say (Unstable) and include installing the latest version of go.

Don't use numbers for none sequential items.  Like install latest or top of tree.